doc/api/fs.md

```diff
@@ -1519,6 +1519,10 @@ changes:
   - version: v2.3.0
     pr-url: https://github.com/nodejs/node/pull/1845
     description: The passed `options` object can be a string now.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: The `open`, `read`, `close` allow overriding the used `fs`
+    implementation.
```

Comment:
The line wrap here for `implementation` is breaking the build. Can't line wrap on the in-line YAML metadata.
```suggestion
```

Reply:
It is strange: lines are wrapped in the `description` field all over the file. Maybe the issue is aligning of the wrapped lines? They are usually aligned with the first char after the `description: ` fragment.